mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb

```diff
@@ -274,6 +274,30 @@ func TestAccComputeSecurityPolicy_withRateLimitWithRedirectOptions(t *testing.T)
 	})
 }
 
+<% unless version == 'ga' -%>
+func TestAccComputeSecurityPolicy_withRateLimit_withEnforceOnKeyConfigs(t *testing.T) {
+	t.Parallel()
+
+	spName := fmt.Sprintf("tf-test-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckComputeSecurityPolicyDestroyProducer(t),
+		Steps: []resource.TestStep{
```

Comment:
Can more tests be added to update and remove enforce_on_key_configs? 
Example: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/tests/resource_compute_security_policy_test.go.erb#L80-L116

Reply:
Hello! I remade the test, I only put the update step, because the remove test got this error: 

![image](https://user-images.githubusercontent.com/120479960/213257579-894d9040-bd53-4cdb-a2b2-82c407e00f10.png)

I don't know if this error is internal or external.

Reply:
I don't see any different between testAccComputeSecurityPolicy_withRateLimitOptions_withEnforceOnKeyConfigs_withRedirectOptionsUpdate and testAccComputeSecurityPolicy_withRateLimitOptions_withEnforceOnKeyConfigs. 


Also, I cannot tell what happened for the error of removal, as I cannot see the removal test.

Reply:
Hello there! Sorry for delay. 

I was trying to create an `enforceOnKeyConfig` update and removal test, but it didn't work, every removal or update failed. So, I had an idea: I created two tests, one with only one `enforceOnKeyConfig` item using redirect options and other with multiple `enforceOnKeyConfig` items. I took as an example the `TestAccComputeSecurityPolicy_withRateLimitOptions`, there are two test with `enforceOnKey` with two examples, but there is no update or removal test.